ids_validator/checks/generic/athena.py

```diff
@@ -195,3 +192,34 @@ def get_athena_partitions_name(cls, athena_schema) -> list:
             for partition in partitions
         ]
         return names
+
+    @classmethod
+    def normalize_path_name(cls, path_name):
+        """
+        Weird-partition!@name -> weird_partition_name
+        @fileId -> fileid
+        project.name -> project_name
+        """
+        normalized_path = re.sub("[^A-Za-z0-9]+", "_", path_name)
+        normalized_path = re.sub("[_]+", "_", normalized_path)
+        normalized_path = normalized_path.strip("_")
```

Comment:
It looks like only a leading underscore is invalid: https://github.com/tetrascience/ts-lambda-core/blob/development/lambdas/dlToAthena/normalizers/index.js#L18
```suggestion
        normalized_path = normalized_path.lstrip("_")
```

Reply:
fixed